.buildkite/pipeline.yml

```diff
@@ -22,17 +20,21 @@ steps:
 
   - wait
 
-  - group: ":hammer_and_wrench: Binary builds"
+  - group: ":hammer_and_wrench: Build binaries"
     steps:
     - name: ":{{matrix.os}}: Build {{matrix.os}} {{matrix.arch}} binary"
-      command: ".buildkite/steps/build-binary.sh {{matrix.os}} {{matrix.arch}}"
-      key: build-binary
-      artifact_paths: "pkg/*"
+      artifact_paths: "dist/test-splitter-*"
       plugins:
-        docker-compose#v4.14.0:
-          config: .buildkite/docker-compose.yml
-          cli-version: 2
-          run: build
+        docker#v5.11.0:
+          image: goreleaser/goreleaser:v2.1.0
```

Comment:
Oh I see you already have a matrix here. How is this working? I thought goreleaser pro only supported target builds

Reply:
I use `--single-target` option that will build the binary only for specific GOOS and GOARCH. The non-pro version however doesn't support split & merge release.
By the way, this build is for the CI pipeline, we don't use this for the release because we have to rebuild the binaries with correct version.

Reply:
Oh right yes okay